storage/innobase/include/buf0buf.h

```diff
@@ -2297,6 +2297,10 @@ struct buf_pool_t {
   running. Protected by flush_state_mutex. */
   os_event_t no_flush[BUF_FLUSH_N_TYPES];
 
+  /** This is event is set when there is no ongoing invalidation of the
+   * buff_pool_t. */
```

Comment:
We can improve the comment a bit:

This event is always set at startup, so LRU threads do not wait for this event. Before invalidating bufferpool, this event is reset, so the next LRU batch flushing will wait for the event. Bufferpool invalidation needs LRU flushing to be stopped